client/my-sites/stats/stats-purchase/stats-purchase-single-item.tsx

```diff
@@ -72,21 +74,30 @@ const StatsCommercialPurchase = ( {
 	redirectUri,
 }: StatsCommercialPurchaseProps ) => {
 	const translate = useTranslate();
+	const isOdysseyStats = config.isEnabled( 'is_running_in_jetpack_site' );
+
+	const commercialTitle = isOdysseyStats
```

Comment:
I guess this is by the design, but it doesn't make much sense to me to show different names here. We could go with just 'Jetpack Stats Commercial' here. What do you think?

Reply:
Of course, I just don't want to miss anything on the design.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/81359/commits/67ab8660930607fcb55788b1baf7c7edfa9cf132.